### DISCUSSION TITLE
Duplicate Alias Entries after Upgrade to 4.6.2-Beta?

### DISCUSSION THREAD

**Discussion by PureFallen** [simple-login/app#1247] [Self-hosting question] [answered]
I recently upgraded an self-hosted instance of SimpleLogin from 3.4.0 to 4.6.2-Beta using the instructions in [upgrade.md](https://github.com/simple-login/app/blob/4.6.2-beta/docs/upgrade.md) .

Since this upgrade, some aliases appear multiple times in the alias listing. The selection of which are affected seems to be completely random, or might be connected to prior Send/Receive History. 

![image](https://user-images.githubusercontent.com/68307987/186911179-537e58c9-14a9-46a0-80c4-f8f44beebd0d.png)

When looking through the issues, i couldn't find anything related to it. But i am generally unsure if this is a bug, or intended behavior to display Alias History instead of listening all Aliases available?
A quick look into the Postgres DB Table `alias` shows that the entries are not duplicated there. The Alias Counter at the very top is also working. When trying to expand the information of an Alias using the "More" button, it always expands this information on the first occurrence of that Alias. Really weird.

And of course, i wanted to check the possibility of a personal error before throwing in an Issue Report there...
Did anyone experience something similar so far? If so, did i miss out on any Issues/Discussions related to it?

**Comment by nguyenkims** [comment, marked as answer]
@PureFallen I suspect this was due to a change in `email_log` table. Can you run the following command and let me know if the issue is solved? Please replace `myuser` by your Postgres user.

```bash
docker exec -it sl-db psql -U myuser simplelogin
update email_log set alias_id=(select alias_id from contact where contact.id = email_log.contact_id);
exit
```

**Reply by PureFallen** [reply]
I made sure to create a backup of the database before, but the command worked. Thanks a lot!

Any suspicions as to what caused the problem? Did I possibly even do something wrong in the upgrade process?

**Reply by nguyenkims** [reply]
There was a model change that requires this database manual update, we should have put that into the upgrade doc when the change was added ...
